pkg/client/controller/applyconfiguration/common/v1beta1/algorithmsetting.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022 The Kubeflow Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by applyconfiguration-gen. DO NOT EDIT.
```

Comment:
This is generated by code-gen >= 1.27 ?

Reply:
Yes, that's right. The applyconfiguration allows us to use [SSA](https://kubernetes.io/docs/reference/using-api/server-side-apply/) easily.